lib/KeyStore.js

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+/**
+ * Each instance of this API is associated with a single data hub
+ */
+export class KeyStore {
+  constructor({hub}) {
+    this.hub = hub;
+  }
+
+  /**
+   * Performs initialization (ensures required indexes are created).
+   */
+  init() {
+    // FIXME: enable
+    // this.hub.ensureIndex({attribute: []});
+  }
+
+  /**
+   * Gets a key by its ID.
+   *
+   * @param {string} id - The key ID.
+   */
+  async get({id}) {
+    const keyInfo = await this.hub.get({id});
+    // TODO: use keyInfo to determine some provider/plugin to use to
```

Comment:
When I wrote this TODO, the idea was that some key class would be returned by this API.  But then I got jammed up on determining which layer of abstraction is responsible for determining what KMS/Plugin etc should be used to instantiate such a class.

At this point, I'm inclined to think that this functionality does not exist here, but at some higher level, and that this layer is concerned simply with JSON blobs containing key material.